packages/dbadmin/src/tools/database.ts

```diff
@@ -50,10 +50,11 @@ const init = async (config: PostgresConfig, sqlFile: string) => {
     }
     return true
 }
-
-const command = process.argv[2]
+//
+let command = process.argv[2]
 let sqlFile = ""
 let envFile: PostgresConfig | null = null
+command = "create"
```

Comment:
This will always set `command` to have the value `"create"`: probably not what you want?

Reply:
No, this is going to be removed completely.